docs/datadog_ktx_module.md

```diff
@@ -0,0 +1,49 @@
+# Datadog Kotlin Extensions package
+
+Provides useful Kotlin extensions on top of our core `dd-sdk-android` SDK.
+
+**Note**: This package is an extension of the main package, so add both dependencies into your gradle file.
+
+## Setup
+```conf
+repositories {
+    maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+}
+
+dependencies {
+    implementation "com.datadoghq:dd-sdk-android:x.x.x"
+    implementation "com.datadoghq:dd-sdk-android-ktx:x.x.x"
+}
+```
+
+### Extensions
+
+1. Span extension methods:
```

Comment:
Here you're just listing sample code, but I think we do need a couple of lines to explain what it does.